dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/Config14Tests.java

```diff
@@ -40,7 +40,7 @@ public class Config14Tests extends FATServletClient {
     public static final String SERVER_NAME = "Config14Server";
 
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP50, MicroProfileActions.MP33, MicroProfileActions.LATEST);
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60, MicroProfileActions.MP33, MicroProfileActions.LATEST);
```

Comment:
LATEST has changed meaning.  Do we need MP41 and MP50 both covered?

Reply:
I'm confused by LATEST, I'm not sure why it's still MP41 when MP50 has been out for a while and when I looked at existing tests, many are running against LATEST and MP50, so they'd all need updated as well.

For that reason I haven't changed LATEST in this PR, and unless we can work out a way for this situation to not happen again, I'd advocate for removing it altogether.

For this particular class, I didn't think it was necessary to run against both MP60 and MP50.